src/forms/FormFieldRadio/FormFieldRadio.stories.tsx

```diff
@@ -120,7 +119,7 @@ export const KitchenSink = (): ReactElement => {
 					helperText: "Helper text",
 					instructionText: "Instruction text",
 				},
-			] as FieldDef<FormFieldRadioDef>[],
+			] as FieldDef[],

```

Comment:
For consistency this should go in line 83 next to `conts fields: FieldDef[]...`. Same in line 45 with the playground fields.